extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js

```
@@ -0,0 +1,16 @@
+define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
+	'ext.wikia.adEngine.utils.adLogicZoneParams',
+	'wikia.instantGlobals'
+], function (zoneParams, instantGlobals) {
+	'use strict';
+
+	var placementsMap = instantGlobals.wgAdDriverAppNexusBidderPlacementsConfig;
+
+	function getPlacement(skin) {
+		return placementsMap[skin][zoneParams.getVertical()];
```

Let's just check and throw error if the WF variable is not defined.

---

done 👍